src/main/jbake/content/blog/2017/05/25/jaeger-contributions.adoc

```diff
@@ -0,0 +1,31 @@
+= Kubernetes and OpenShift Templates for Jaeger
+Pavol Loffay
+2017-5-25
+:icons: font
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, apm, tracing, jaeger
+
+I am happy to announce that we made easier deploying Jaeger on Kubernetes clusters. With more
+ongoing contributions to Jaeger project we set up new https://github.com/jaegertracing[Jegertracing]
```

Comment:
Instead of "With more...", it might be better to say "This work has been carried out in a new github organization (ref) established to contain contributions for the Jaeger ecosystem." - we shouldn't say "we" as it was a joint decision to set this up.